docs/api.rst

```diff
@@ -942,6 +956,22 @@ Timeshift a :ref:`Stream` by *n* milliseconds. ::
 
 Delaying a :ref:`Stream` timeshifts all the events by the same amount. It doesn't change the time *between* events.
 
+.. _withLocalTime:
+
+withLocalTime
+`````````````
+
+.. code-block:: haskell
+
+  withLocalTime :: Offset -> Stream a -> Stream a
+
+Create a Stream with localized :ref:`Time` values, whose origin (i.e., time 0) is at the specified Time on the :ref:`Scheduler` used to the Stream is observed with :ref:`runEffects` or :ref:`run`.
```

Comment:
> Scheduler used to the Stream

something maybe missing here ?

Reply:
Yes. That is not a complete sentence.

Reply:
In the code comment for ` withLocalTime`, the following text is found:

> Create a stream with its own local clock
> This transforms time from the provided scheduler's clock to a stream-local clock (which starts at 0), and then *back* to the scheduler's clock before propagating events to sink.  In other words, upstream sources will see local times, and downstream sinks will see non-local (original) times.

Reply:
I think in this case, the code comment provides some extra detail on how withLocalTime works, which may not be appropriate for user api docs.  On the other hand, the last sentence in the comment may be useful in the docs.

From your comment, I'm not sure what you're suggesting.  Could you clarify?  Thanks!